include/cucumber-cpp/internal/hook/HookRegistrar.hpp

```diff
@@ -22,6 +22,7 @@ class CUCUMBER_CPP_EXPORT CallableStep {
 
 class CUCUMBER_CPP_EXPORT Hook {
 public:
+    virtual ~Hook() {}
```

Comment:
Could you add a comment before every virtual destructor you add for this purpose then? Something like:
> Silence false positive Visual C++ warning C4265 requiring virtual destructors for every class that has virtual functions